pyproject.toml

```diff
@@ -4,3 +4,7 @@ requires = [
     "setuptools_scm >= 3.1.0, <4",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.black]
+target-version = ["py36"]
+skip-string-normalization = true
```

Comment:
Think whether to include string normalization or not
Think whether we should set the line length to 119
Add a line